scss/style.scss

```diff
@@ -0,0 +1,1292 @@
+@font-face {
+    font-family: OpenSans;
+    src: url(font/OpenSans/OpenSans-Bold.ttf);
+}
+
+body {
+    margin: 0px;
+    width: 100%;
+}
+
+.container {
+    margin-left: 10%;
+    margin-right: 10%;
+    width: 80%;
+}
+
+.header {
+
+    &__top {
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+        width: 100%;
+        height: 100px;     
+
+        &__logo {
+            font-family: OpenSans;
+            font-size: 25px;
+            color: #212121;
+        }   
+
+        &__menu a {
+            font-family: OpenSans;
+            font-size: 19px;
+            letter-spacing: 0.05em;
+            color: #212121;
+            text-decoration: none;
+            margin-right: 15px;
+        }
+    }
+
+    &__content {
+        width: 100%;
+        height: 520px;
+        position: relative;
+        display: flex;
+        justify-content: center;
+        background: url('/img/header__content.jpg');           
+    
+        &__shadow {
+            position: absolute;
+            width: 100%;
+            height: 100%;
+            background-color: rgba(0, 0, 0, 0.7);
+        }
+
+        &__title {
+            position: absolute;
+            font-family: OpenSans;
+            font-size: 60px;
+            letter-spacing: 0.05em;
+            color: #ffffff;
+            margin-top: 127px;
+        }
+
+        &__subtitle {
+            position: absolute;
+            font-family: OpenSans;
+            font-size: 22px;
+            letter-spacing: 0.05em;
+            color: #ffffff;
+            margin-top: 232px;
+        }
+
+        &__buttons {
+            display: flex;
+            position: absolute;
+            width: 50%;
+            margin-top: 305px;
+            justify-content: center;
+        }
+
+        &__button {
+            font-family: Roboto;
+            font-size: 20px;
+            line-height: 22px;
+            font-weight: 700;
+            letter-spacing: 0.05em;
+            color: #ffffff;
+            border: 2px solid white;
+            margin-right: 25px;
+            width: 150px;
+            height: 22px;
+            padding: 20px 20px;
+            text-align: center;
+
+            &:hover {
+                background-color: #00897b;
+                border: 2px solid #00897b;
+            }
+
+            &:last-child {
+                margin-right: 0px; 
+            }
+        }
+    }
+}
+
+.main {
+
+    &__name {
+        display: flex;
+        width: 100%;
+        flex-direction: column;
+        margin: 100px 0 110px 0;
+        text-align: center;
+
+        &__title {
+            font-family: OpenSans;
```

Comment:
As I see, you use `OpenSans` and some another code uses one more times. So, I guess u need to write `mixin` for this cases with specified variables

Reply:
done